### ISSUE TITLE
Description planets \ moons

### ISSUE THREAD

**Issue by BlesseNtumble** [micdoodle8/Galacticraft#1557]
Actually, it is a defect or how it should be?
 And if a defect, please make the possibility of writing descriptions for add-ons.

![1](http://firepic.org/images/2015-04/08/81z37uagbciz.png)

**Comment by radfast** [comment]
It's a defect / incomplete.

**Comment by Nick-77ab2** [comment]
ermagerd this means half life 3 confirmed right?

**Comment by Nick-77ab2** [comment]
jk, so it's kind of like an Easter egg?

**Comment by dj3520** [comment]
>It's a defect / incomplete.

**Comment by Nick-77ab2** [comment]
i thought he meant incomplete like it was still being worked on when the build was released, which is why i said Easter egg as a joke

**Comment by BlesseNtumble** [comment]
When to add a description of the planets?

**Comment by BlesseNtumble** [comment]
Need descriptions =)

**Comment by radfast** [comment]
Yes.  I'm looking at this.  It's not easy, there is only limited space in the GUI.

**Comment by BlesseNtumble** [comment]
Hmm, I think you can do this:

gui.(name of the selected planet \ moon \ star).desc = description
And most likely better to increase the information window

**Comment by radfast** [comment]
This is a pretty GUI.  But I will point out, the planet is not square...

**Comment by BlesseNtumble** [comment]
Well, instead of round planets use the registered image for the planet \ moon.
If necessary, I can make a sketch GUI window.

**Comment by radfast** [comment]
> If necessary, I can make a sketch GUI window.

Yes please, sounds great.

**Comment by BlesseNtumble** [comment]
@radfast 
![1](http://5.firepic.org/5/images/2016-01/29/2p0qmrldt6hf.png)
![2](http://5.firepic.org/5/images/2016-01/29/s62lhr6mx2sl.png)
![3](http://5.firepic.org/5/images/2016-01/29/64fw5nmv4mcf.png)

**Comment by radfast** [comment]
nice job, thank you it helps a lot.

1. I agree with you about a summary description top right.  As much information as possible should come from in-game data, not ```.desc``` text.  I might even add more in-game data.

2. We might look at just a symbol or a tick for breathable atmosphere, less text is better.

3. Some of these descriptions - especially ```gui.[name].information.desc``` will be too long - I don't want to include so much text in the main ```.lang``` files.  So I'm thinking a new .lang file in a folder named ```galacticraftcatalog```.  Translating galacticraftcatalog will be optional and the translation style can be different - maybe the translators can find information about the planet in their own language on Wikipedia or somewhere else, or maybe they want to write a shorter or longer text in their language.

I am also thinking about merging all galacticraftmoon ```.lang``` files into galacticraftcore, and merging galacticraftmars and galacticraftasteroids ```.lang``` files into one.

4.  I will add a celestial body boolean field ".undiscovered".  If true, no description will be shown, only an undiscovered message.  Add-on authors can create in-game conditions which switch that to false.  Do you think this undiscovered data needs to be saved **per player**?  Or do you think if one player discovers a planet, the whole world knows immediately?

**Comment by BlesseNtumble** [comment]
Did you make a configuration for enable \ disable save per player.

**Comment by radfast** [comment]
OK we can do it like this:
```CelestialBody.setExplored(EntityPlayerMP)```
```CelestialBody.setExploredAll()```

**Comment by radfast** [comment]
I'm planning to add something like this in 1.8.9

**Comment by BlesseNtumble** [comment]
![1](https://pp.userapi.com/c626228/v626228755/4caaa/BmI5xXi_Vlw.jpg)
